index.html

```diff
@@ -39,7 +41,88 @@ <h1 class="salute">I'm Alexandra. Glad to see you!</h1>
                 </li>
             </ul>
         </div>
-    </div>
-
+        </div>
+    </section>
+    <div id="work">
+        <section id="tonic1">
+            <div class="card">
+                <a class="card_image" href="#"><img src="tonic1_image.svg" alt="tonic1"></a>
+                <div class="card_bottom">
+                    <h1 class="card_title">Tonic</h1>
+                    <ul class="card_ul">
+                        <li><p class="canopy">CANOPY</p></li>
+                        <li><img class="counter" src="Counter.svg" alt="counter"></li>
+                        <li><p class="text_li">Back End Dev</p></li>
+                        <li><img class="counter" src="Counter.svg" alt="counter"></li>
+                        <li><p class="text_li">2015</p></li>
+                    </ul>
+                    <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="categories">
+                        <li><p>HTML</p></li>
+                        <li><p>CSS</p></li>
+                        <li><p>JavaScript</p></li>
+                    </ul>
+                    <a class="see_project_button" href="#">See project</a>
+                </div>
+                
+            </div>
+        </section>
+        <section id="stories1">
+            <div class="card">
+                stories1
+            </div>
+        </section>
+        <section id="tonic2">
+            <div class="card">
+                tonic2
+            </div>
+        </section>
+        <section id="stories2">
+            <div class="card">
+                stories2
+            </div>
+        </section>

```

Comment:
- [ ] Kindly complete all cards for the work section and try to match the Figma design. :+1: